src/vs/workbench/common/events.ts

```diff
@@ -229,15 +234,15 @@ export class CommandEvent extends Event {
 }
 
 /**
- * Viewlet events are emitted when a viewlet opens or closes in the sidebar.
+ * Composite events are emitted when a composite opens or closes in the sidebar or panel.
  */
-export class ViewletEvent extends Event {
-	public viewletId: string;
+export class CompositeEvent extends Event {
+	public compositeId: string;
 
-	constructor(viewletId: string, originalEvent?: any) {
+	constructor(compositeId: string, originalEvent?: any) {
```

Comment:
@isidorn I think here we should also send around the actual instance of Composite that was opened or closed because then one can simply do an instance of check to find out if the composite was a viewlet or panel.